src/backend/looped/vehicle/vehicle_abilities.cpp

```diff
@@ -0,0 +1,102 @@
+#include "backend/looped_command.hpp"
+#include "util/vehicle.hpp"
+#include "util/misc.hpp"
+
+#include <vehicle/CVehicleModelInfo.hpp>
+
+namespace big
+{
+	enum eVehicleAbilities : int16_t
```

Comment:
This belongs in either gta/enums if it's from GTA Itself, or core/enums if you generated it yourself for other reasons.

Reply:
It will only ever be used here, so no it's fine where it is imo.

Reply:
That wasn't a request.